plugins/modules/packaging/os/portage.py

```diff
@@ -113,6 +113,12 @@
     choices: [ "web", "yes", "no" ]
     type: str
 
+  getbinpkgonly:
+    description:
+      - Merge only packages specified at PORTAGE_BINHOST in make.conf
```

Comment:
```suggestion
      - Merge only packages specified at C(PORTAGE_BINHOST) in C(make.conf).
```

Reply:
Also, you should mention here (and for the other option) that they are mutually exclusive.